data_safe_haven/resources/workspace/workspace.cloud_init.mustache.yaml

```
@@ -37,6 +37,8 @@ write_files:
       popd
 
 mounts:
+  # Mount ephemeral storage at resource instead of default /mnt
+  - [ephemeral0, /mnt/resource]
```

```suggestion
  - [ephemeral0, /mnt/scratch]
```

A suggestion for the mount name. "scratch" feels much more natural to me for a (relatively) fast disk where data may be lost.